### ISSUE TITLE
Robustness to linear quantization

### ISSUE THREAD

**Issue by wzds2015** [iamhankai/ghostnet.pytorch#28]
I've tested Ghost Module recently on light weight Unet model and found it can provide speed gain with 1-2% accuracy drop. But after linear quantization, the accuracy is much lower. Wonder whether author has any research on this?

**Comment by iamhankai** [comment]
Quantization for mobilenet series is more difficult than ordinary CNNs (https://github.com/tensorflow/tensorflow/tree/r1.14/tensorflow/contrib/quantize). Try not to quantize the depthwise convolution?
